src/components/index.ts

```diff
@@ -1 +1,2 @@
 export { default } from './Navbar/Navbar';
+export * from './inputField/index'
```

Comment:
Please follow the same export scheme that `Navbar` follows.

For example:
```suggestion
export { default as Navbar} from './Navbar/Navbar';
export { default as PasswordField } from './PasswordField/PasswordField';
export { default as TextField } from './TextField/TextField';
```
This will work if you implement my folder structure recommendation and use `default` exports for your components.